code/modules/jobs/job_types/brig_physician.dm

```diff
@@ -27,7 +27,7 @@
 	species_outfits = list(
 		SPECIES_PLASMAMAN = /datum/outfit/plasmaman/brig_physician
 	)
-	biohazard = 15 //still deals with the sick and injured, just less than a medical doctor
+	biohazard = 25 //still deals with the sick and injured, just less than a medical doctor
```

Comment:
these changes are undocumented

Reply:
this is the variable that determines dormant disease chance

Reply:
It seems Kat was confused with a single CL line that might have to be two lines for two changes.